internal/move2kube/planner.go

```diff
@@ -49,9 +49,6 @@ func CreatePlan(inputPath string, prjName string, interactive bool) plantypes.Pl
 	} else {
 		containerizer.InitContainerizers(p.Spec.Inputs.RootDir, nil)
 	}
-	if len(selectedTranslationPlanners) == 0 {
```

Comment:
why remove this?
we can change it to warning/debug at least

Reply:
Yep.